app/uk/gov/hmrc/merchandiseinbaggage/service/CalculationService.scala

```diff
@@ -69,7 +70,7 @@ class CalculationService @Inject()(mibConnector: MibConnector)(implicit ec: Exec
 
   def thresholdAllowance(declaration: Declaration)(implicit hc: HeaderCarrier): Future[ThresholdAllowance] = {
     import declaration._
-    val totalGoods = declarationGoods.goods ++ paidAndNotRequired(amendments)
+    val totalGoods = declarationGoods.goods ++ DeclarationView.validAmendments(declaration).flatMap(_.goods.goods)
```

Comment:
if you wanted to use `DeclarationView` then you can replace above with `DeclarationView.allGoods(....)` call